repository-data/webfiles/src/main/resources/site/freemarker/common/macro/sections/quoteSection.ftl

```diff
@@ -0,0 +1,56 @@
+<#ftl output_format="HTML">
+<#include "../../../include/imports.ftl">
+
+<#-- @ftlvariable name="section" type="uk.nhs.digital.website.beans.Quote" -->
+
+<#macro quoteSection section>
+    <div class="quote-box" aria-label="Quotation">
+       <div itemscope itemtype="https://schema.org/Quotation">
+         <div class="quote-text">
+           <span itemprop="text"><@hst.html hippohtml=section.quote contentRewriter=gaContentRewriter/></span>
+         </div>
+       <div class="quote-box-role">
+		     <#if section.role?has_content && section.person?has_content && section.organisation?has_content>
```

Comment:
Just wondering if this could be optimised to reduce some of the repetition of various elseif statements.  Happy to help take a look.